src/notion/blocks.ts

```diff
@@ -185,3 +196,25 @@ export function equation(value: string): Block {
     },
   };
 }
+
+export function callout(
+  text: RichText[] = [],
+  emoji: EmojiRequest = '👍',
+  color: ApiColor = 'default',
+  children: Block[] = [],
+): Block {
+  return {
+    object: 'block',
+    type: 'callout',
+    callout: {
+      rich_text: text.length ? text : [richText('')],
+      icon: {
+        type: 'emoji',
+        emoji,
+      },
+      // @ts-expect-error Typings are not perfect
```

Comment:
You're right! I've opened this issue in the Notion client repo: https://github.com/makenotion/notion-sdk-js/issues/575

In the meanwhile, we can change that to this:
```suggestion
      // @ts-expect-error See https://github.com/makenotion/notion-sdk-js/issues/575
```